### ISSUE TITLE
running rfswarm fails with the following error ModuleNotFoundError: No module named 'requests'

### ISSUE THREAD

**Issue by ryanrosello-og** [damies13/rfswarm#123]
I'm running windows 11, Python 3.11.1, pip 22.3.1

**To Reproduce**
run `pip install rfswarm-manager`

After installation, try and launch the manager via `rfswarm`

**Expected behavior**
Manager launched successfully

**Logs**

```
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Install the latest PowerShell for new features and improvements! https://aka.ms/PSWindows

Loading personal and system profiles took 805ms.
 ~  pip install rfswarm-manager
Collecting rfswarm-manager
  Downloading rfswarm_manager-1.0.0-py3-none-any.whl (90 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 90.9/90.9 kB 2.5 MB/s eta 0:00:00
Collecting configparser
  Downloading configparser-5.3.0-py3-none-any.whl (19 kB)
Collecting HTTPServer
  Downloading httpserver-1.1.0-py2.py3-none-any.whl (7.8 kB)
Requirement already satisfied: pillow in c:\users\ryanr\appdata\local\programs\python\python311\lib\site-packages (from rfswarm-manager) (9.4.0)
Collecting psutil
  Downloading psutil-5.9.4-cp36-abi3-win_amd64.whl (252 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 252.5/252.5 kB 5.2 MB/s eta 0:00:00
Requirement already satisfied: pip>=21,>=22 in c:\users\ryanr\appdata\local\programs\python\python311\lib\site-packages (from rfswarm-manager) (22.3.1)
Collecting matplotlib
  Downloading matplotlib-3.6.3-cp311-cp311-win_amd64.whl (7.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 7.2/7.2 MB 6.0 MB/s eta 0:00:00
Collecting docopt
  Downloading docopt-0.6.2.tar.gz (25 kB)
  Preparing metadata (setup.py) ... done
Collecting contourpy>=1.0.1
  Downloading contourpy-1.0.7-cp311-cp311-win_amd64.whl (162 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 163.0/163.0 kB 4.9 MB/s eta 0:00:00
Collecting cycler>=0.10
  Downloading cycler-0.11.0-py3-none-any.whl (6.4 kB)
Collecting fonttools>=4.22.0
  Downloading fonttools-4.38.0-py3-none-any.whl (965 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 965.4/965.4 kB 4.7 MB/s eta 0:00:00
Collecting kiwisolver>=1.0.1
  Downloading kiwisolver-1.4.4-cp311-cp311-win_amd64.whl (55 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.4/55.4 kB 3.0 MB/s eta 0:00:00
Requirement already satisfied: numpy>=1.19 in c:\users\ryanr\appdata\local\programs\python\python311\lib\site-packages (from matplotlib->rfswarm-manager) (1.24.1)
Collecting packaging>=20.0
  Downloading packaging-23.0-py3-none-any.whl (42 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 42.7/42.7 kB 2.2 MB/s eta 0:00:00
Collecting pyparsing>=2.2.1
  Downloading pyparsing-3.0.9-py3-none-any.whl (98 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 98.3/98.3 kB 5.5 MB/s eta 0:00:00
Collecting python-dateutil>=2.7
  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 5.2 MB/s eta 0:00:00
Requirement already satisfied: six>=1.5 in c:\users\ryanr\appdata\local\programs\python\python311\lib\site-packages (from python-dateutil>=2.7->matplotlib->rfswarm-manager) (1.16.0)
Installing collected packages: docopt, python-dateutil, pyparsing, psutil, packaging, kiwisolver, HTTPServer, fonttools, cycler, contourpy, configparser, matplotlib, rfswarm-manager
  DEPRECATION: docopt is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for docopt ... done
Successfully installed HTTPServer-1.1.0 configparser-5.3.0 contourpy-1.0.7 cycler-0.11.0 docopt-0.6.2 fonttools-4.38.0 kiwisolver-1.4.4 matplotlib-3.6.3 packaging-23.0 psutil-5.9.4 pyparsing-3.0.9 python-dateutil-2.8.2 rfswarm-manager-1.0.0

[notice] A new release of pip available: 22.3.1 -> 23.0
[notice] To update, run: python.exe -m pip install --upgrade pip
 ~  rfswarm
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Scripts\rfswarm.exe\__main__.py", line 4, in <module>
  File "C:\Users\ryanr\AppData\Local\Programs\Python\Python311\Lib\site-packages\rfswarm_manager\rfswarm.py", line 37, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
 ryanr   ~                                                                                in pwsh at 18:36:49 
```

I then install `pip install requests` manually and the Manager eventually appeared

**Comment by damies13** [comment]
Thanks for the bug report 👍🏼, looks like I need to include requests in the prerequisites of the pip package.

**Comment by damies13** [comment]
Fix Added to release v1.0.1 branch
